dev_tools/nginx/README.md

```diff
@@ -4,48 +4,63 @@
 
 - Docker (18.03+ for Mac/Windows, 20.04+ for Linux)
 - KF-portal-UI is running locally on port 3000
-- `portal-qa.includedcc.org` and/or `portal.includedcc.org` added to hosts file
+- `portal-qa.kidsfirstdrc.org` and/or `portal.kidsfirstdrc.org` added to hosts file
+
+## Setup
+
+- Open a terminal and exec `sudo vim /etc/hosts` or `sudo nano etc/hosts`
+- Add both/one of the following line
```

Comment:
line(s). 
Could you add that if one wants to access the added urls (for instance, I want to reach to QA but `portal-qa.kidsfirstdrc.org` is used locally) he needs to remove/comment it out from hosts file.